src/lsm/forest_table_iterator.zig

```diff
@@ -84,10 +81,14 @@ pub fn ForestTableIteratorType(comptime Forest: type) type {
                                 forest.tree_for_id_const(tree_id),
                                 iterator.level,
                             )) |table| {
-                                return .{
+                                return table.encode(.{
+                                    .tree_id = tree_id,
                                     .level = iterator.level,
-                                    .table = table.encode(tree_id),
-                                };
+                                    // Dummy event, doesn't really mean anything in this context.
+                                    // (We are reusing the schema's TableInfo type since it is
+                                    // shared by all Tree types.)
+                                    .event = .insert,
+                                });
```

Comment:
Could we use instead `.event = .reserved` for cases where a dummy event is required?


Also here:
https://github.com/tigerbeetle/tigerbeetle/blob/aceb8f736526e00834ce660356cc1bf6b936c4c7/src/vsr/replica.zig#L7931